Tools/python/Samples.py

```diff
@@ -826,188 +1428,341 @@ def _set_signal_stuff(sample):
+#     DataSample('SingleMuon2018D', '/SingleMuon/Run2018D-12Nov2019_UL2018-v8/AOD'),
+#     DataSample('EGamma2018A', '/EGamma/Run2018A-12Nov2019_UL2018-v2/AOD'),
+#     DataSample('EGamma2018B', '/EGamma/Run2018B-12Nov2019_UL2018-v2/AOD'),
+#     DataSample('EGamma2018C', '/EGamma/Run2018C-12Nov2019_UL2018-v2/AOD'),
+#     DataSample('EGamma2018D', '/EGamma/Run2018D-12Nov2019_UL2018-v8/AOD'),
+# ]
 
-DisplacedJet_data_samples_2018 = [
-    DataSample('DisplacedJet2018A', '/DisplacedJet/Run2018A-15Feb2022_UL2018-v1/AOD'),
-    DataSample('DisplacedJet2018B', '/DisplacedJet/Run2018B-15Feb2022_UL2018-v1/AOD'),
-    DataSample('DisplacedJet2018C', '/DisplacedJet/Run2018C-15Feb2022_UL2018-v1/AOD'),
-    DataSample('DisplacedJet2018D', '/DisplacedJet/Run2018D-15Feb2022_UL2018-v1/AOD'),
+BTagCSV_data_samples_2018 = [
+    DataSample('BTagCSV2018A', '/BTagCSV/Run2018A-15Feb2022_UL2018-v2/MINIAOD'),
+    DataSample('BTagCSV2018B', '/BTagCSV/Run2018B-15Feb2022_UL2018-v1/MINIAOD'),
+    DataSample('BTagCSV2018C', '/BTagCSV/Run2018C-15Feb2022_UL2018-v1/MINIAOD'),
     ]
 
-# FIXME we could just put them here and drop from the JetHT list
-BTagCSV_data_samples_2018 = [
-    #DataSample('JetHT2018A', '/JetHT/Run2018A-15Feb2022_UL2018-v2/AOD'),
-    #DataSample('JetHT2018B', '/JetHT/Run2018B-15Feb2022_UL2018-v1/AOD'),
-    #DataSample('JetHT2018C', '/JetHT/Run2018C-15Feb2022_UL2018-v1/AOD'),
-    #DataSample('JetHT2018D', '/JetHT/Run2018D-15Feb2022_UL2018-v1/AOD'),
+DisplacedJet_data_samples_2018 = [
+    DataSample('DisplacedJet2018A', '/DisplacedJet/Run2018A-15Feb2022_UL2018-v1/MINIAOD'),
+    DataSample('DisplacedJet2018B', '/DisplacedJet/Run2018B-15Feb2022_UL2018-v1/MINIAOD'),
+    DataSample('DisplacedJet2018C', '/DisplacedJet/Run2018C-15Feb2022_UL2018-v1/MINIAOD'),
+    DataSample('DisplacedJet2018D', '/DisplacedJet/Run2018D-15Feb2022_UL2018-v1/MINIAOD'),
     ]
 
-MuonEG_data_samples_2018 = [
-    DataSample('MuonEG2018A', '/MuonEG/Run2018A-15Feb2022_UL2018-v1/AOD'),
-    DataSample('MuonEG2018B', '/MuonEG/Run2018B-15Feb2022_UL2018-v1/AOD'),
-    DataSample('MuonEG2018C', '/MuonEG/Run2018C-15Feb2022_UL2018-v1/AOD'),
-    DataSample('MuonEG2018D', '/MuonEG/Run2018D-15Feb2022_UL2018-v1/AOD'),
+Lepton_data_samples_2018 = [
+    DataSample('SingleMuon2018A', '/SingleMuon/Run2018A-UL2018_MiniAODv2-v3/MINIAOD'),
+    DataSample('SingleMuon2018B', '/SingleMuon/Run2018B-UL2018_MiniAODv2-v2/MINIAOD'),
+    DataSample('SingleMuon2018C', '/SingleMuon/Run2018C-UL2018_MiniAODv2-v2/MINIAOD'),
+    DataSample('SingleMuon2018D', '/SingleMuon/Run2018D-UL2018_MiniAODv2-v3/MINIAOD'),
+    # DataSample('EGamma2018A', '/EGamma/Run2018A-UL2018_MiniAODv2-v1/MINIAOD'),
+    # DataSample('EGamma2018B', '/EGamma/Run2018B-UL2018_MiniAODv2-v1/MINIAOD'),
+    # DataSample('EGamma2018C', '/EGamma/Run2018C-UL2018_MiniAODv2-v1/MINIAOD'),
+    # DataSample('EGamma2018D', '/EGamma/Run2018D-UL2018_MiniAODv2-v2/MINIAOD'),
```

Comment:
Should we re-enable electrons? Also in 2016. It looks like things were previously split by electron vs muon data samples, and we could return to that if it would help....
@pkotamnives @abbywarden @duquale